src/app-layout/__integ__/toolbar-tooltips/app-layout-toolbar-drawer-trigger-tooltip.test.ts

```diff
@@ -0,0 +1,165 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import createWrapper from '../../../../lib/components/test-utils/selectors/index';
+import { drawerIds as drawerIdObj } from '../../../../lib/dev-pages/pages/app-layout/utils/drawer-ids';
+import { VISIBLE_MOBILE_TOOLBAR_TRIGGERS_LIMIT } from '../constants';
+import { setupTest } from '../utils';
+
+const wrapper = createWrapper().findAppLayout();
+const drawerIds = Object.values(drawerIdObj);
+
+describe('refresh-toolbar', () => {
+  //using a theme variable below that will be set when iterating over ['refresh-toolbar', 'refresh']
+  const theme = 'refresh-toolbar';
+  const mobileDrawerTriggerIds = drawerIds.slice(
+    0,
+    VISIBLE_MOBILE_TOOLBAR_TRIGGERS_LIMIT + (theme === 'refresh-toolbar' ? 1 : 0)
+  );
+
+  describe.each(['desktop', 'mobile'] as const)('%s', size => {
+    const drawerIdsToTest = size === 'mobile' ? mobileDrawerTriggerIds : drawerIds; //toolbarDrawerIds;
+    const firstDrawerTriggerSelector = wrapper.findDrawerTriggerById(drawerIdsToTest[0]).toSelector();
+    const expectedFirstDrawerTriggerTooltipText = 'Security'; //must match drawerItems[0].ariaLabels.drawer from /lib/dev-pages/pages/app-layout/utils/drawers
```

Comment:
What is the point of declaring this variable, whereas you can use`drawerItems[0].ariaLabels.drawer` ?

Reply:
unable to import the drawerItems. I tried investigating and I do not want to changes the typescript config for all the code for this as the object contains tsx files methods